app/src/main/java/org/mozilla/fenix/settings/autofill/AutofillSettingFragment.kt

```diff
@@ -293,6 +295,11 @@ class AutofillSettingFragment : BiometricPromptPreferenceFragment() {
         context.settings().incrementSecureWarningCount()
     }
 
+    private val startForResult = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) {
+            result: ActivityResult ->
+        pinRequestResult(PIN_REQUEST, result.resultCode)
```

Comment:
just a passing by comment: if we go with the extension function, `pinRequestResult` function could be removed

Reply:
Not sure if there's a clean way to do this.